src/Model.hs

```diff
@@ -48,24 +72,18 @@ update c = do
   let weekday = (date ^. (_localDay . mondayWeek . _mwDay)) - 1
   let theDate = formatTime defaultTimeLocale "%F" date
   rest <- sequence
-    [ fetchMenu "21f31565-5c2b-4b47-d2a1-08d558129279" theDate
-      <&> ( Restaurant
-              "K\229rrestaurangen"
-              "http://carbonatescreen.azurewebsites.net/menu/week/karrestaurangen/21f31565-5c2b-4b47-d2a1-08d558129279"
-          . transformMenu Swe
-          )
-    , fetchMenu "3d519481-1667-4cad-d2a3-08d558129279" theDate
-      <&> ( Restaurant
-              "Express Johanneberg"
-              "http://carbonatescreen.azurewebsites.net/menu/week/johanneberg-express/3d519481-1667-4cad-d2a3-08d558129279"
-          . transformMenu Swe
-          )
-    , fetchMenu "3ac68e11-bcee-425e-d2a8-08d558129279" theDate
-      <&> ( Restaurant
-              "S.M.A.K."
-              "http://carbonatescreen.azurewebsites.net/menu/week/smak/3ac68e11-bcee-425e-d2a8-08d558129279"
-          . transformMenu Swe
-          )
+    [ fetchMenu Swedish "21f31565-5c2b-4b47-d2a1-08d558129279" theDate
```

Comment:
Perhaps we can lift out the guid and name in the url too and let fetchMenu know about the base url. (in the name of reducing duplication :))

Reply:
That's a very nice idea, I'll start thinking about a solution to this after I'm done with the logging.